content/sql/shop-project/_index.md

```diff
@@ -5,8 +5,8 @@ flavours:
 - none
 prerequisites:
   hard:
-  - topics/solo-learn/sql/4-challenges
-  - docker/intro-to-docker/
+    - topics/solo-learn/sql/4-challenges
```

Comment:
is the additional spaces here intentional?

Reply:
It was in an attempt to shift the bullet points to the right but it had no affect and I forgot to undo that change.